tests/test_keybindings.py

```diff
@@ -46,13 +46,17 @@ def test_simple_keybinding_single_mod(mod: str, key: str) -> None:
     assert int(as_full_kb) == int(kb)
 
 
-def test_simple_keybinding_multi_mod():
+@pytest.mark.skipif(sys.platform != "darwin", reason="mac only")
```

Comment:
these skips don't seem right to me... You should absolutely be able to represent the *concept* of `"win+shift+A"` on a mac:

```py
In [1]: from app_model.types import SimpleKeyBinding

In [2]: SimpleKeyBinding.from_str("win+shift+A")
Out[2]:
SimpleKeyBinding(
    ctrl=False,
    shift=True,
    alt=False,
    meta=True,
    key=<KeyCode.KeyA: 18>
)
```


why would we lose that?

Reply:
i don't see how this is related to your Shift+Return issue.   This was always meant to be platform agnostic at this point

Reply:
but...

```
In [3]: SimpleKeyBinding.from_str("win")
Out[3]: SimpleKeyBinding(ctrl=False, shift=False, alt=False, meta=False, key=<KeyCode.UNKNOWN: 0>)
```

Reply:
ahh... I see, it's the ending with a modifier issue again,  ok, for that we need to look not at SimpleKeyBinding, but at KeyCode I think.  lemme take a look

Reply:
Even on windows this the win keyCode seem to be UNKNOWN

Reply:
yeah, this is the line then that needs consideration:

https://github.com/pyapp-kit/app-model/blob/08d0fbd6274ebdcf0d2634dc0bf7f8a9b4446004/src/app_model/types/_keys/_keybindings.py#L79

specifically... we need the logic that maps "win" and "cmd" to meta:

https://github.com/pyapp-kit/app-model/blob/08d0fbd6274ebdcf0d2634dc0bf7f8a9b4446004/src/app_model/types/_keys/_keybindings.py#L289-L292

to be added to the `KeyCode.from_string` logic:

https://github.com/pyapp-kit/app-model/blob/08d0fbd6274ebdcf0d2634dc0bf7f8a9b4446004/src/app_model/types/_keys/_key_codes.py#L588-L591

Reply:
right, because if we're talking about single keys (not key combos) then there is only the `Meta` key... 

https://github.com/pyapp-kit/app-model/blob/08d0fbd6274ebdcf0d2634dc0bf7f8a9b4446004/src/app_model/types/_keys/_key_codes.py#L93

so just need to update `KeyCode.from_string()` to map `win` and `cmd` to that

Reply:
just FYI, in general, there isn't any "statefulness" in the key representations that pays any attention to what OS you're on until the last minute, when the keybinding actually gets registered at runtime:

https://github.com/pyapp-kit/app-model/blob/08d0fbd6274ebdcf0d2634dc0bf7f8a9b4446004/src/app_model/registries/_keybindings_reg.py#L54-L69


all the rest of the keybinding stuff is OS-agnostic